tests/conftest.py

```diff
@@ -0,0 +1,14 @@
+import pytest
+import os
+
+
+@pytest.fixture
+def chdir():
```

Comment:
Could you add a docstring and comments to this to make this fixture more followable?